src/social-network-provider/facebook.com/tasks/pasteIntoPostBox.ts

```diff
@@ -89,7 +89,7 @@ export async function pasteIntoPostBoxFacebook(
             if (e) e.style.display = 'none'
         }
         // Prevent Custom Paste failed, this will cause service not available to user.
-        if (element.innerText.indexOf(text) === -1 && 'value' in element && element.value.indexOf(text) === -1)
+        if (element.innerText.indexOf(text) === -1 || ('value' in element && element.value.indexOf(text) === -1))
```

Comment:
Property 'value' does not exist on type 'HTMLDivElement | HTMLTextAreaElement'. So `'value' in element` always returns false.

Reply:
oh nice catch